bin/verifyBytecodes.ts

```diff
@@ -0,0 +1,116 @@
+import { execSync } from "node:child_process";
+import fs from "node:fs";
+import process from "node:process";
+import { providers } from "ethers";
+
+const broadcastFiles = [
+  "broadcast/MU01-00-Create-Proxies.sol",
+  "broadcast/MU01-01-Create-Nonupgradeable-Contracts.sol",
+  "broadcast/MU01-02-Create-Implementations.sol",
+];
```

Comment:
Let's replace this by listing all folders in the broadcast dir.

Reply:
Actually, after reading this, I had an idea, let's actually take the "MU01" as an argument, remove the address list as an argument, and then just compile a list of all the contracts created in all the broadcasts files that match: `broadcast/${UPGRADE}-*/run-latests.json`, and then output a list to the terminal:
```
Contract Address                               Contract Name            Bytecode matches
-----------------------------------------------------------------------------------------
0xB881aF21C5A9ff8e8d5E4C900F67F066C6CB7936     BreakerBoxProxy          ✅
0xFF9a3da00F42839CD6D33AD7adf50bCc97B41411     BiPoolManagerProxy       ✅

```